.travis.yml

```diff
@@ -1,30 +1,16 @@
-dist: xenial
-sudo: required
-services:
-  - docker
```

Comment:
this is to allow your tests to run docker not to run the tests in docker so there's no point to enable it